### ISSUE TITLE
Extension news integration

### ISSUE THREAD

**Issue by heikohaensge** [DMKEBUSINESSGMBH/typo3-t3socials#8]
Is there already a documented solution to integrate the news extension ( https://extensions.typo3.org/extension/news/ ) in t3socials?

**Comment by hannesbochmann** [comment]
No, but you can integrate it easily on your own. @see https://github.com/DMKEBUSINESSGMBH/typo3-t3socials/blob/master/Documentation/DeveloperManual/Index.md
